public/docs/advanced/truffle-with-metamask.md

```diff
@@ -1,69 +1,91 @@
 
-Currently, MetaMask only supports Google's Chrome browser, but support for Firefox and others is planned for the future.
+![MetaMask](/docs/img/metamask.png)
 
-To install MetaMask, go to the [Chrome Web Store](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn) and click the + ADD TO CHROME button.
+For development with Truffle this means we can use our dapp the same way users will interact with it on a live network.
 
-# Using MetaMask with the TestRPC
+## Installing MetaMask
 
-With our front-end ready to be used and MetaMask installed, we're ready to see our Dapp in all its glory. Before diving in though, we'll need to make sure we're checking for MetaMask's `web3` instance and the extension itself is configured properly with the TestRPC.
+Currently, MetaMask only supports the Chrome browser, but support for Firefox and others is planned for the future.
 
-## Detecting MetaMask's Web3 Injection
+To install MetaMask, go to the [Chrome Web Store](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn) and click the **Add to Chrome** button.
 
-MetaMask injects its own `web3` instance, so we'll want to make sure we're checking for that. After the window as loaded perform the following check:
+
+## Using MetaMask with Truffle Develop
+
+Truffle Develop runs an in-memory blockchain that is used for testing purposes. It runs on `localhost:9545`.
+
+With our front-end ready to be used and MetaMask installed, we're ready to see our dapp in all its glory. Before diving in though, we'll need to make sure we're checking for MetaMask's `web3` instance and the extension itself is configured properly with Truffle Develop.
+
+### Detecting MetaMask's web3 injection
+
+MetaMask injects its own `web3` instance, so we'll want to make sure we're checking for that. After the window has loaded perform the following check:
 
 ```javascript
 // Is there is an injected web3 instance?
 if (typeof web3 !== 'undefined') {
   App.web3Provider = web3.currentProvider;
   web3 = new Web3(web3.currentProvider);
 } else {
-  // If no injected web3 instance is detected, fallback to the TestRPC.
-  App.web3Provider = new web3.providers.HttpProvider('http://localhost:8545');
+  // If no injected web3 instance is detected, fallback to Truffle Develop.
+  App.web3Provider = new web3.providers.HttpProvider('http://localhost:9545');
   web3 = new Web3(App.web3Provider);
 }
 ```
 
-## Using the Browser Extension
+### Interacting with MetaMask
+
+To use Truffle Develop with MetaMask, click the MetaMask icon in your browser and this screen will appear:
```

Comment:
This line caught me off guard because it seems similar to this line? https://github.com/trufflesuite/trufflesuite.com/pull/15/files#diff-76e148b4ebf55ab2e2914fe77db0be20R26

Perhaps different wording or a clarification could be added.

Reply:
Sorry, I don't think I follow. Did you mean that "Using MetaMask with Truffle Develop" sounds too close to "Interacting with MetaMask" or something else?

Reply:
You have a section titled "Using MetaMask with Truffle Develop", and then you start this section, titled differently, with "To use Truffle Develop with Metamask". Because of their similarities it originally structure me as an error.